cmd/joy/execute.go

```diff
@@ -0,0 +1,186 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/nestoca/survey/v2"
+
+	"github.com/nestoca/joy/internal/config"
+
+	"github.com/spf13/cobra"
+)
+
+func NewExecuteCmd() *cobra.Command {
+	var list bool
+	var jsonOutput bool
+	cmd := &cobra.Command{
+		Use:     "execute",
+		Aliases: []string{"exec", "x"},
```

Comment:
`e` is not used, is your intent to use it for `environment`?

```suggestion
		Aliases: []string{"exec", "e"},
```

Reply:
Yeah, `e` will be used for `environments`, for one-letter command lovers like yourself! 😄